### ISSUE TITLE
Incorrect argument in `set_cycles_texture`

### ISSUE THREAD

**Issue by StandingPadAnimations** [Moo-Ack-Productions/MCprep#531]
### Did the issue happen after restarting Blender?

- [X] Yes, the issue indeed continued after restarting Blender

### Check known issues

- [X] I have checked known issues, and my problem is different

### What happened?

MCprep 3.5.2 has a bug where we use the wrong keyword argument here:
https://github.com/Moo-Ack-Productions/MCprep/blob/5aaee32a88cab2ff7dc97037980596600379a899/MCprep_addon/materials/generate.py#L351-L352

`use_extra_passes` is now `extra_passes`

### What steps are needed to produce the bug?

Use texture swap

### What version of MCprep are you using?

MCprep 3.5.2

### What version of Blender are you using?

All of them

**Comment by TheDuckCow** [comment]
It's also the top 1 and 2 spots by quite a margin in the send-to-developer report (just showing the third to show how distant it is, but is not related):

<img width="695" alt="Screen Shot 2024-01-08 at 10 54 16 AM" src="https://github.com/Moo-Ack-Productions/MCprep/assets/2958461/b6f64329-06dd-416e-b20d-9cf4701551a3">

Full error for future onlookers, thanks for being quick to find a fix:

```
482, in execute
 res += generate.set_texture_pack(mat, folder, self.useExtraMaps)
 File "<addon_path>/generate.py", line 351, in set_texture_pack
 _ = set_cycles_texture(
 TypeError: set_cycles_texture() got an unexpected keyword argument 'use_extra_passes'
```

**Comment by StandingPadAnimations** [comment]
Fixed in #532
